config/environments/test.rb

```diff
@@ -32,6 +32,8 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
+  # Required for testing user invitations
+  config.action_mailer.default_url_options = { host: config.wcrs_back_office_url, protocol: "http" }
```

Comment:
Same as above :)

Reply:
Yes, tests fail without this setting. This follows what we had to do for WEX as well.